### PR TITLE
Rework the stylesheets

### PR DESCRIPTION
**Pull request by knightburton**
Rework the stylesheets based on the #81 issue.
- Apply nested properties on the rules.
- Apply parent selectors on the rules.
- Split the main file into small maintainable pieces.

IoT.jsCode-DCO-1.0-Signed-off-by: Imre Kiss kissi.szeged@partner.samsung.com

### CODE REVIEW

File: src/app/style/bright/sidenav-extra.scss [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+        &.disabled {
+          opacity: $default-opacity;
+          &:hover,
+          &:hover input { cursor: not-allowed; }
+
+          label:hover {
+            color: inherit !important;
+            cursor: inherit !important;
+            color: #fff;
```
Comment (daeyeon) [inline src/app/style/bright/sidenav-extra.scss]: cursor and color are duplicated in this block. I think we can use the inherent value for them.

File: src/app/style/bright/app.scss [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+body {
+  background-color: $b-background-color;
+  word-wrap: normal;
+  height: 100%;
+  font-size: $font-size-3;
+  line-height: 1.4em;
+}
+
+h1, h2, h3, h4 {
```
Comment (daeyeon) [inline src/app/style/bright/app.scss]: I guess that `single line per selector` is your implicit convention. I guess this is only one violating it. My suggestion :
```
h1,
h2,
h3,
h4 {
```

**Comment by knightburton** [comment]
@daeyeon I've updated the PR based on your comments.

**Comment by yichoi** [comment]
LGTM

**Pull request merged**